pkg/network/ovn_kubernetes.go

```diff
@@ -568,6 +600,7 @@ func bootstrapOVN(conf *operv1.Network, kubeClient client.Client) (*bootstrap.Bo
 			OVNKubernetesConfig:     ovnConfigResult,
 			PrePullerDaemonset:      prePullerDS,
 			Platform:                platformType,
+			FlowsConfig:             flowsconfig.Bootstrap(kubeClient),
```

Comment:
Isn't this always going to be nil at startup? (I'm assuming the NOO starts up after the CNO)

Reply:
This is expected to be nil most of the times, as it will be only not nil when the flow tracing is enabled. If it  is nil, the `renderOVNFlowsConfig` function just ignores it and then the `ovnkube-node` instances will be rendered with the usual configuration.